index.html

```diff
@@ -142,13 +168,13 @@
   <h1>Calendar</h1>
 </div>
 <p class="centered">
-<iframe src="https://www.google.com/calendar/embed?src=96mpu88v2d3onqkp6ph9865p1c%40group.calendar.google.com&amp;ctz=Asia/Taipei" style="border: 0" width="800" height="600"></iframe>
+<iframe src="https://www.google.com/calendar/embed?src=96mpu88v2d3onqkp6ph9865p1c%40group.calendar.google.com&amp;ctz=Asia/Taipei&amp;mode=agenda" style="border: 0" width="800" height="600"></iframe>
 
 <div class="page-header" id="irc">
   <h1>Web IRC / #taipeihack</h1>
 </div>
 <p class="centered">
-<iframe src="https://webchat.freenode.net?randomnick=1&channels=%23taipeihack&prompt=1&uio=OT10cnVlde" width="800" height="600"></iframe>
+<iframe src="https://webchat.freenode.net?randomnick=1&amp;channels=%23taipeihack&amp;prompt=1&amp;uio=OT10cnVlde" width="800" height="600"></iframe>
```

Comment:
did your editor mess up the link? Most "&" became "&amp;" which should not not be (see [query strings](http://en.wikipedia.org/wiki/Query_string#Web_forms))